lib/ui/pointer.dart

```diff
@@ -38,21 +38,49 @@ enum PointerChange {
   up,
 
   /// A pan/zoom has started on this pointer.
+  ///
+  /// This type of events will always have kind [PointerDeviceKind.trackpad].
   panZoomStart,
 
   /// The pan/zoom on this pointer has updated.
+  ///
+  /// This type of events will always have kind [PointerDeviceKind.trackpad].
   panZoomUpdate,
 
   /// The pan/zoom on this pointer has ended.
+  ///
+  /// This type of events will always have kind [PointerDeviceKind.trackpad].
   panZoomEnd,
 }
 
 /// The kind of pointer device.
 enum PointerDeviceKind {
   /// A touch-based pointer device.
+  ///
+  /// The most common case is a touch screen.
+  ///
+  /// When the user is operating with a trackpad on iOS (for example, Magic
+  /// Keyboard on iPad), clicking will also dispatch events with kind [touch] if
+  /// `Info.plist` is not present or returns NO for
```

Comment:
Nit: I think this sounds slightly better but it's up to you, not a big difference.

"or returns" => "or it returns"

Reply:
Thanks for finding this, I actually wrote it wrong. I've changed it to:
```dart
  /// [...] if `UIApplicationSupportsIndirectInputEvents` is not present in `Info.plist`
  /// or returns NO.
```
Does it sound right now?

Reply:
Sounds good to me, thanks!